app/api/src/docker/cadquery/cadquery.js

```diff
@@ -1,53 +1,55 @@
 const { runCQ } = require('./runCQ')
 const middy = require('middy')
 const { cors } = require('middy/middlewares')
+const AWS = require('aws-sdk')
+const {
+  makeHash,
+  checkIfAlreadyExists,
+  getObjectUrl,
+  loggerWrap,
+  storeAssetAndReturnUrl,
+} = require('../common/utils')
 
-// cors true does not seem to work in serverless.yml, perhaps docker lambdas aren't covered by that config
-// special lambda just for responding to options requests
-const preflightOptions = (req, _context, callback) => {
-  const response = {
-    statusCode: 204,
-    headers: {
-      'Access-Control-Allow-Origin': '*',
-      'Access-Control-Allow-Methods': 'POST',
-      'Access-Control-Allow-Headers': '*',
-    },
-  }
-  callback(null, response)
-}
```

Comment:
This was me handling CORS preflight endpoints directly because I couldn't get the config working. Turns out it had something to do with binary media types which we don't need since we're not returning the asset directly now.